craco.config.js

```diff
@@ -55,15 +55,16 @@ module.exports = function({ env }) {
                       "ClientV2",
                       "ModelManagerV5",
                       "PingerV1",
+                      "JIMMV1",
```

Comment:
This is the only real change to this file. The rest are just `prettier` changes.